modules/board/delete.php

```diff
@@ -16,7 +16,7 @@
 
 // Delete board
 } else {
-    match ($stepParameter) {
+    match ((int)$stepParameter) {
```

Comment:
What do you think about adding
```
$stepParameter = $_GET['step'] ?? 0;
$stepParameter = (int) $stepParameter;
```

on top of the file? This would ensure the parameter is always an integer and not only in this particular match case. I can't think of a scenario, where a string would be beneficial here.

Reply:
Now that you forced me to think about it again:
I would say the best way would be to use `$request->query->getInt('step',0)`.
As that enforces a default value, does the type casting and refactors already to utilize functionality of HttpFoundation.
I would be so lazy to drop that directly into the match statement, but I guess the cleaner way would be to initialize a variable on top. 
And yes, all of those step variables are integers...which is also we may want to change or at least define as constants